docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -109,7 +109,99 @@ var flexLayout = new FlexLayout();
 flexLayout.WrapType = FlexLayout.FlexWrapType.NoWrap;
 layoutView.Layout = flexLayout;
 ```
+There are six properties that affect the specified child view.
 
+> [!NOTE]
+> These properties are added in Tizen 6.0.
```

Comment:
@dasg34
How about using a common note instead of individual notes? 

> [!NOTE]
> The `FlexAlignmentSelf`, `FlexPositionType` ... ... properties are added in Tizen 6.0.

We can remove individual notes.